ItemChanger/Events/GameEvents.cs

```diff
@@ -10,13 +9,15 @@ public static class GameEvents
     /// <summary>
     /// Called after persistent items reset.
     /// </summary>
-    public static event Action? OnPersistentUpdate;
+    public static event Action OnPersistentUpdate { add => _OnPersistentUpdateSubscribers.Add(value); remove => _OnPersistentUpdateSubscribers.Remove(value); }
+    private static readonly List<Action> _OnPersistentUpdateSubscribers = [];
```

Comment:
Would prefer camelCase for fields throughout 

```suggestion
    private static readonly List<Action> onPersistentUpdateSubscribers = [];
```